conftest.py

```diff
@@ -32,7 +32,7 @@ def integration_config(request, tmpdir_factory):
 
   db_path_new = str(tmpdir_factory.mktemp('tmpdir_factory').join("icubam.db"))
 
-  exit_code = call(['sqlite3', str(db_path), '.backup {db_path_new}'])
+  exit_code = call(['sqlite3', str(db_path), f'.backup {db_path_new}'])
```

Comment:
Wouldn't it be safer to escape, e.g. `f".backup '{db_path_new}'"`, something like that? For example if the path contains spaces.

Reply:
Thanks. Quoting path could be useful indeed.